### DISCUSSION TITLE
How to read and display DB data on a page using HTML

### DISCUSSION THREAD

**Discussion by Pingu-Pox** [requarks/wiki#3955] [Help / Questions]
I wanted to do fun things for my internal wiki's homepage, such as:

- Show count of total pages in DB
- Link to latest pages
- Link to pages with lots of hits/edits.

I'd want to do this within the HTML of my home page, and have that info refresh with the page. I have pgAdmin4 running alongside the below and am able to connect to the DB and browse the tables. I was able to build the queries that I would need to run, but haven't the foggiest idea on how to do it within the html on a page. I did some googling and some people use PHP to query the PostgreSQL DB from the webpage, is that a possibility here too? I took a course like 4 years ago on PHP so it wouldn't be too hard for me to relearn.

Wiki.js Version: 2.5.201
PostgreSQL: 11.11

Both Wiki.js and PostgreSQL are running from docker containers using the same docker compose file from the wiki.js documentation (obviously not using these exact environment variables):

`version: "3"
services:

  db:
    image: postgres:11-alpine
    environment:
      POSTGRES_DB: wiki
      POSTGRES_PASSWORD: wikijsrocks
      POSTGRES_USER: wikijs
    logging:
      driver: "none"
    restart: unless-stopped
    volumes:
      - db-data:/var/lib/postgresql/data

  wiki:
    image: requarks/wiki:2
    depends_on:
      - db
    environment:
      DB_TYPE: postgres
      DB_HOST: db
      DB_PORT: 5432
      DB_USER: wikijs
      DB_PASS: wikijsrocks
      DB_NAME: wiki
    restart: unless-stopped
    ports:
      - "80:3000"

volumes:
  db-data:`

**Comment by FelipeAscensio** [comment]
Hello Pingu

Do you find a solution?

I also want to do that it
